.github/workflows/checks.yml

```diff
@@ -5,38 +5,37 @@ on:
     branches:
       - master
     paths-ignore:
-      - 'docs/**'
+      - "docs/**"
   pull_request:
     paths-ignore:
-      - 'docs/**'
+      - "docs/**"
 jobs:
-
   lint:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.11"]
```

Comment:
don't we want to run this with the python versions that flytekit supports?

Reply:
This is for the lint, we don't need to run it with different Python versions.